Client/Configuration/Fennec.xcconfig

```diff
@@ -7,6 +7,20 @@ MOZ_BUNDLE_DISPLAY_NAME = Fennec ($(USER))
 // Bundle Identifier
 MOZ_BUNDLE_ID = org.mozilla.ios.Fennec
 
+
 // The suffix of the app group identifier used for sharing data across
 // extensions and keychain
 MOZ_GROUP_SUFFIX = org.mozilla.ios.Firefox
+
+// Flag to indicate if we want to include the debug settings bundle or not
+INCLUDE_SETTINGS_BUNDLE = YES
+
+// Adjust SDK Variables
+ADJUST_ENVIRONMENT =
+ADJUST_APP_TOKEN = 
+
+// Leanplum SDK Variables
+LEANPLUM_ENVIRONMENT = development
```

Comment:
Yes I need help with this, let's do this in a separate branch or something. Contact me about this!